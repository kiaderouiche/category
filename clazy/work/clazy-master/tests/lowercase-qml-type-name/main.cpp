#include <QtCore/QObject>
#include <QtCore/QString>
#include <QtQml/QQmlEngine>

struct Foo : public QObject {};

void test()
{
     qmlRegisterType<Foo>("App", 1, 0, "Foo"); // OK
     qmlRegisterType<Foo>("App", 1, 0, "foo"); // Warn
}
