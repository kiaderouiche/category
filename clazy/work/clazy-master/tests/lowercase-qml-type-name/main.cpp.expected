lowercase-qml-type-name/main.cpp:10:40: warning: QML types must begin with uppercase [-Wclazy-lowercase-qml-type-name]
